ClassLibrary/ClassLibrary/DepartmentReaders.cs

```diff
@@ -48,9 +48,20 @@ public int CountBorrowedBooksWithISBN(String ISBN)
             return BorrowedBooksWithISBN(ISBN).Count;
         }
 
-        public HashSet<Book> BorrowedBooksWithISBN(String ISBN)
+        public List<BorrowedBook> BorrowedBooksWithISBN(String ISBN)
         {
-            return new HashSet<Book>(Readers.SelectMany(x => x.BorrowedBooks.Where(y => y.ISBN == ISBN)));
+            return Readers.SelectMany(x => x.BorrowedBooks.Where(y => y.book.ISBN == ISBN)).ToList();
+        }
+
+        public TimeSpan GetTimeWhenFreeBook(List<BorrowedBook> borrowedBooksISBN)
```

Comment:
The return type should be `DateTime`, not `TimeSpan`. it's much easier to understand that "the book is available on June 23rd" than "the book is available in 16 days, 18 hours and 44 minutes".

Reply:
ok